### ISSUE TITLE
Bug: Account value is doubled-ish for investments

### ISSUE THREAD

**Issue by ahatzz11** [maybe-finance/maybe#1517]
**Describe the bug**
When viewing an investment account synced through plaid, the total value of the account seems to be roughly rouble what it should be. 

My hunch here is that Maybe is doing some weird math of 'value of investments + cash balance'. When I go to the holdings page I see holdings, but if I go to cash I see a accurate account value amount of cash as well. The total value seems to be double the cash amount.

**To Reproduce**
1. Sync an investment account (like wealthfront) via plaid
2. Go to accounts page
3. See total is ~double what it should be

**Expected behavior**
I expect all values in the app to be correct.

**What version of Maybe are you using?**
Hosted

**What operating system and browser are you using?**
MacOS + Chrome

**Screenshots / Recordings**
![CleanShot-000678 2024-12-07 at 21 38 12@2x](https://github.com/user-attachments/assets/15a8ccf3-e9ca-49f8-838e-140c905ea89d)

**Comment by zachgoll** [comment]
@ahatzz11 I have a big overhaul of Investment accounts coming today/tomorrow that handles brokerage cash a lot better and won't double things.

The reason this is happening is because Plaid is sending us a "Cash Holding" _in addition_ to the stated "brokerage cash" for the account.  We're treating "holdings" and "brokerage cash" as 2 separate entities in Maybe, which causes the doubling.

The overhaul I'm doing will specifically look for the "Cash Holding" that Plaid sends us and will back that out of the total holdings calculation, thus removing the double-count here.

**Comment by ahatzz11** [comment]
> @ahatzz11 I have a big overhaul of Investment accounts coming today/tomorrow that handles brokerage cash a lot better and won't double things.
> 
> The reason this is happening is because Plaid is sending us a "Cash Holding" _in addition_ to the stated "brokerage cash" for the account.  We're treating "holdings" and "brokerage cash" as 2 separate entities in Maybe, which causes the doubling.
> 
> The overhaul I'm doing will specifically look for the "Cash Holding" that Plaid sends us and will back that out of the total holdings calculation, thus removing the double-count here.

Awesome! Looking forward to it.

**Comment by zachgoll** [comment]
As of the latest commit, this doubling should be resolved.

That said, we still have several things to work through for Plaid investment accounts, so those will be addressed in subsequent issues/PRs.  A few off the top of my head that still need to be addressed:

- Some users will still show a "holding" of `CUR:USD`.  We'll need to perform a DB update to remove those across all accounts and properly handle `CUR:USD` "transactions" that Plaid sends us.
- If an investment account has an options holdings, we are not yet setup to handle those

In other words, keep raising issues as you find them, but we're still working through a lot on the investment side of things right now :)
